### PR TITLE
Quiz questions page

### PR DESCRIPTION
**Pull request by PrashanthreddyAinala**
questions and result page updated

screencapture-127-0-0-1-5500-pages-quiz-html-2022-03-27-10_04_01

### CODE REVIEW

File: pages/result.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    <title>True Quiz</title>
```
Comment (khushijohri2001) [inline pages/result.html:3]: Not mandatory but you can use meta tags for responsive design, author, and description. Its a good practice

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  --white: #ffffff;
+}
+
+body {
+  margin: 0px;
+  font-family: "Inter", sans-serif;
+}
+
+p {
+  margin: 0px;
```
Comment (khushijohri2001) [inline styles/styles.css:13]: Instead of 0px you can give the value without the unit too. Simply margin: 0;

**Review by khushijohri2001 — APPROVED** [review]
Other than that, Clean code!! There are no extra spaces and lines, well-aligned. All-in-all great job👍
Just a suggestion, it would be better if you could align the whole content in the center or give more colors to it.

File: styles/styles.css (hunk 2 above)
Comment (PrashanthreddyAinala) [reply styles/styles.css:13]: thankyou for this review sure I will update these things.

**Review by Veekshith-Thoram — APPROVED** [review]

**Pull request merged**